pkl-codegen-kotlin/src/main/kotlin/org/pkl/codegen/kotlin/KotlinCodeGenerator.kt

```diff
@@ -513,6 +533,18 @@ class KotlinCodeGenerator(
     else generateRegularClass()
   }
 
+  private fun TypeSpec.Builder.ensureKSerializable(): TypeSpec.Builder {
+    if (!options.implementKSerializable) {
+      return this
+    }
+
+    val spec = AnnotationSpec.builder(Serializable::class).build()
```

Comment:
We can avoid adding a dependency on kotlinx serialization by referencing kotlinx serialization by string:

```suggestion
    val spec = AnnotationSpec
      .builder(ClassName("kotlinx.serialization", "Serializable"))
      .build()
```